src/pages/HomePage/HomePage.js

```diff
@@ -0,0 +1,43 @@
+import NavBar from "../../Common-Components/NavBar";
+import React, { useEffect } from "react";
+import styles from "./HomePage.module.less";
+import { Link } from "react-router-dom";
+import Loader from "../../Common-Components/Loader/Loader";
+
+function HomePage(props) {
+  const { fetchingBlogList, blogList, isFetchingBlogList } = props;
+
+  useEffect(() => {
+    fetchingBlogList();
+  }, [fetchingBlogList]);
+
+  return (
+    <>
+      <NavBar />
+
+      {isFetchingBlogList === true ? (
```

Comment:
isFetchingBlogList===true is not required only isFetchingBlogLis is enough

Reply:
Resolved